File: model/src/main/java/net/artux/pda/model/news/ArticleModel.java
package net.artux.pda.model.news;

import java.time.Instant;
import java.util.List;

import lombok.Data;

@Data
public class ArticleModel {

    private String id;
    private String title;
    private String image;
    private String url;
    private List<String> tags;
    private String description;
    private Instant published;

}

File: model/src/main/java/net/artux/pda/model/news/ArticleModel.kt
package net.artux.pda.model.news

import java.time.Instant

class ArticleModel {
    var id: String? = null
    var title: String? = null
    var image: String? = null
    var url: String? = null
    var tags: List<String>? = null
    var description: String? = null
    var published: Instant? = null
    var likes: Int = 0
    var comments: Int = 0
}